addon/appModules/notepad++/addonGui.py

```diff
@@ -42,7 +42,7 @@ def __del__(self):
 			gui.mainFrame.sysTrayIcon.preferencesMenu.RemoveItem(self.prefsMenuItem)
 			#If we die, so did the app. If the user relaunches, so should we.
 			GuiManager.isMultiInst = False
-		except wx.PyDeadObjectError:
+		except:  # wx.PyDeadObjectError is replaced by RuntimeError as of wx 4
```

Comment:
I already fixed this when I did some addon updates, but I didn't see your PR. I'm sorry!
For future reference, the way you can catch this error is 
except RuntimeError:
or 
except (RuntimeError, FooError):

Doing a plain except is dangerous because it can hide actual errors.